db/migrate/20150708162823_create_fagus_item_types.rb

```diff
@@ -0,0 +1,11 @@
+class CreateFagusItemTypes < ActiveRecord::Migration
+  def change
+    create_table :fagus_item_types do |t|
+      t.string :description, :null => false
+      t.integer :is_active, :default => 0, :null => false
```

Comment:
I was being flexible as we may chose to stop offering a type of item, but we wouldn't want to remove it because of historical data.

Does that make sense?